### DISCUSSION TITLE
How to rename columns in vaex?

### DISCUSSION THREAD

**Discussion by dharik-arsath** [vaexio/vaex#1778] [Q&A]

**Comment by JovanVeljanoski** [comment]
You edited out your description.. was that on purpose or by accident?

**Reply by dharik-arsath** [reply]
sorry, I cannot able to delete that. 

I asked," how to rename columns?", in my case I've to remove all hypen in column names and put underscore there.

**Comment by JovanVeljanoski** [comment]
Like this?
```python
import vaex
df = vaex.example()
df.rename('x', 'new_x')
print(df)
```

Note that the rename operation is done in place.
